src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -345,23 +345,23 @@ void mp::QemuVirtualMachine::start()
     vm_process->write(qmp_execute_json("qmp_capabilities"));
 }
 
-void mp::QemuVirtualMachine::shutdown(bool force)
+void mp::QemuVirtualMachine::shutdown(ShutdownPolicy shutdown_policy)
 {
     std::unique_lock<std::mutex> lock{state_mutex};
 
-    force_shutdown = force;
+    force_shutdown = (shutdown_policy == ShutdownPolicy::Poweroff);
 
```

Comment:
@sharder996 I just noticed that `force_shutdown` value should be set to true later in order to truely pair up with [this line](https://github.com/canonical/multipass/blob/b685dbf331818a030a1c6c290bd6c90eab343b5b/src/platform/backends/qemu/qemu_virtual_machine.cpp#L663),  `check_state_for_shutdown` has a chance to throw and leaves `force_shutdown` to true without actually executing forceful shutdown. Something we overlooked in the silencing error PR. I think the right place to set it might be after line 370, `mpl::log(mpl::Level::info, vm_name, "Killing process");`. What do you think? That makes 

```
            mpl::log(mpl::Level::info, vm_name, "Killing process");
            force_shutdown = true;
```